pydatastructs/graphs/algorithms.py

```diff
@@ -11,6 +11,8 @@
 from pydatastructs.graphs.graph import Graph
 from pydatastructs.linear_data_structures.algorithms import merge_sort_parallel
 from pydatastructs import PriorityQueue
+from copy import deepcopy
+import heapq
```

Comment:
We do have `heapq` implemented in PyDataStructs? I think `PriorityQueue` and `heapq` are the same?